spec/javascripts/surveys-spec.js

```diff
@@ -90,25 +87,109 @@ describe('Surveys', function () {
       expect(GOVUK.cookie(surveys.surveySeenCookieName(defaultSurvey))).toBe('1')
     })
 
-    describe("for a 'url' survey", function () {
-      it('links to the url for a smartsurvey survey with a completion redirect query parameter', function () {
-        surveys.displaySurvey(urlSurvey)
+    describe('for a url survey', function () {
```

Comment:
:+1: - I think having these specs for each survey type is nice and explicit.  Good call.